spec/requests/linking/link_defendant_maat_reference_v2_spec.rb

```diff
@@ -24,7 +24,7 @@
   end
 
   before do
-    allow(ENV).to receive(:fetch).with('LAA_REFERENCES', false).and_return('true')
+    ENV['LAA_REFERENCES'] = 'true'
```

Comment:
Could we keep this as an allow?

Prefer allow because the above implementation will override the ENV['LAA_REFERENCES'] in .env.test and may cause other tests to fail/flicker

Reply:
I see in spec/requests/linking/link_defendant_no_maat_id_spec.rb 
I guess add the allow stubs there too.